go.mod

```diff
@@ -43,3 +42,6 @@ replace github.com/docker/docker => github.com/docker/docker v1.4.2-0.2020022723
 replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.0-pre1.0.20180209125602-c332b6f63c06
 
 replace k8s.io/client-go => k8s.io/client-go v0.0.0-20180806134042-1f13a808da65
+
+// necessary for langserver with vscode
```

Comment:
I'm not sure go keeps comments. If you run `go mod tidy` this goes away and will show up in a different PR.

Reply:
go mod preserves comments:
```
$ tail -2 go.mod
// necessary for langserver with vscode
replace github.com/sourcegraph/go-lsp => github.com/radeksimko/go-lsp v0.0.0-20200223162147-9f2c54f29c9f

$ go mod tidy

$ tail -2 go.mod
// necessary for langserver with vscode
replace github.com/sourcegraph/go-lsp => github.com/radeksimko/go-lsp v0.0.0-20200223162147-9f2c54f29c9f
```